.github/workflows/deleted-files.yml

```
@@ -0,0 +1,38 @@
+name: Deleted files
+
+on:
+  pull_request:
+    branches:
+      master
+
+jobs:
+  list-deleted-files:
+    runs-on: ubuntu-latest
+    name: Detect deleted files
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Get changed docs md files
```

Small style suggestion: maybe add an empty line between each step in the job. It helps with readability when checking out the workflow definition:

```suggestion
      - name: Checkout
        uses: actions/checkout@v3

      - name: Get changed docs md files
```